test/README.md

```diff
@@ -0,0 +1,10 @@
+# Testing considerations
+
+## How to run pytest
+```
+python3 -m pytest
+```
+## Test parsers
+### Facebook Messenger
+
+When adding a new test object in test/facebookMessenger/target.json, make sure that the ```hour``` attribute is in UTC-0. When running pytest, ```hour``` and ```timedate``` will be converted to the users timezone to avoid timezone errors.
```

Comment:
This would rather fit into a CONTRIBUTING.MD document (which we don't have as of now)

Reply:
Should i hold on to this information and add it when CONTRIBUTING.MD is created? Or should I create it in this PR?